senderSampleApp/src/main/java/org/kinecosystem/sampleSenderApp/kindiscover/SendKinService.java

```diff
@@ -15,14 +17,13 @@
 public class SendKinService extends SendKinServiceBase {
 
     @Override
-    public @NonNull
-    KinTransferComplete transferKin(@NonNull String toAddress, int amount, @NonNull String memo) throws KinTransferException {
+    public
+    void transferKin(@NonNull String toAddress, int amount, @NonNull String memo, @NotNull kinTransferCallback callback) {
```

Comment:
You need to provide a way to remove the callback. 
Otherwise we could be calling back the activity listener after the activity has died.